### PR TITLE
TDP-4001: Conference Modify via api

### PR DESCRIPTION
**Pull request by vsmith-bw**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `stop-gap-v2`

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4001.s3-website-us-east-1.amazonaws.com/

File: voice/bxml/callbacks/about.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ We use a 15s request timeout when trying to deliver all callbacks, retrying once
 | [Conference Created](conferenceCreated.md)        | Bandwidth API sends this when a new conference is created.                                                                                                                                                   |
 | [Conference Member Join](conferenceMemberJoin.md) | Bandwidth API sends this when a new member joins a conference.                                                                                                                                               |
 | [Conference Member Exit](conferenceMemberExit.md) | Bandwidth API sends this when a member leaves a conference.                                                                                                                                                  |
+| [Conference Redirect](conferenceRedirect.md)      | Bandwidth API sends this to the application when a POST request is made to the [/conferences/{conferenceId}](../../methods/conferences/postConferencesConferenceId.md) endpoint requesting to modify a conference |
```
Comment (muesono-daitan) [inline voice/bxml/callbacks/about.md:31]: Sadly we have some duplication in the docs, so this list of callbacks also exist in other 2 documents, so you just need to copy this line in `voice/about.md` and in `bxml/about.md` and adjust the link paths accordingly.

File: voice/methods/conferences/postConferencesConferenceId.md [hunk 2]
```diff
@@ -18,6 +18,10 @@ Bandwidth's Voice API leverages Basic Authentication with your Dashboard API Cre
 | Parameter       | Description                                                                     | Mandatory |
 |:----------------|:--------------------------------------------------------------------------------|:----------|
 | status          | (optional) Setting the conference status to `completed` ends the conference.    | No        |
+| redirectUrl     | (optional) The URL to send the Redirect event which will provide new BXML. Not allowed if `state` is `completed`, but required if `state` is `active`    | No        |
+| redirectMethod  | (optional) The HTTP method to use for the request to `redirectUrl` Not allowed if `state` is `completed`    | No        |
+| username        | (optional) The username to send in the HTTP request to `answerUrl` and `diconnectUrl`    | No        |
+| password        | (optional) The password to send in the HTTP request to `answerUrl` and `diconnectUrl`    | No        |
```
Comment (muesono-daitan) [inline voice/methods/conferences/postConferencesConferenceId.md]: `request to 'redirectUrl'`

File: voice/bxml/callbacks/about.md (hunk 1 above)
Comment (CatexB) [reply voice/bxml/callbacks/about.md:31]: Also it needs to go in SUMMARY.md so it shows up in the side nav bar

Comment (vsmith-bw) [reply voice/bxml/callbacks/about.md:31]: thanks! added it to these other pages

File: voice/methods/conferences/postConferencesConferenceId.md (hunk 2 above)
Comment (vsmith-bw) [reply voice/methods/conferences/postConferencesConferenceId.md]: fixed

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4001.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4001.s3-website-us-east-1.amazonaws.com/

**Pull request merged**